main/sketch.cpp

```diff
@@ -270,13 +271,18 @@ void loop() {
         myControllers.processInputs();
     }
 
+    brightness = brightness + fadeAmount;
+    if (brightness <= 0 || brightness >= 255) {
+      fadeAmount = -fadeAmount;
+    }
     // The main loop must have some kind of "yield to lower priority task" event.
     // Otherwise, the watchdog will get triggered.
     // If your main loop doesn't have one, just add a simple `vTaskDelay(1)`.
     // Detailed info here:
     // https://stackoverflow.com/questions/66278271/task-watchdog-got-triggered-the-tasks-did-not-reset-the-watchdog-in-time
 
     //     vTaskDelay(1);
-    delay(150);
+    analogWrite(PIN_LED_FRONT, brightness);
+    delay(30);
```

Comment:
we need to keep this at 150 for now, we will change it later after we are done programming
```suggestion
    delay(150);
```